### ISSUE TITLE
Error Sync Git in ABAP Trail Cloud 

### ISSUE THREAD

**Issue by Th0masMa** [abap2UI5/abap2UI5#710]
Hi,

i cannot sync the Source Code in Eclipse ADT with my Cloud ABAP Trail Account.

![image](https://github.com/abap2UI5/abap2UI5/assets/64013300/61189096-81a5-4d40-b385-565cf7462a8a)

Regards
Thomas

**Comment by larshp** [comment]
as far as I remember, there is a setting in the previous screen for the folder logic?

It must be https://github.com/abap2UI5/abap2UI5/blob/main/.abapgit.xml#L7

**Comment by Th0masMa** [comment]
![image](https://github.com/abap2UI5/abap2UI5/assets/64013300/50d36b90-7aef-4b78-87fc-b7df1e65f954)

**Comment by oblomov-dev** [comment]
go one screen back, i think there is a dropdown where you can change the folder logic.

but i also have a lot of trouble with adt-abapgit, its unfortunately by for not as as stable as the original abapGit. when there are still problems you can try to delete as many a2ui5 artifacts as possible manually before starting the install. or do it the other way around and create the folder logic manually and then try to install the project with the root folder. sometimes this helped me when i had problems.

**Comment by Th0masMa** [comment]
It works. 

![image](https://github.com/abap2UI5/abap2UI5/assets/64013300/9d2bfa9a-93c2-42c2-be07-6265cf03c2f9)
![image](https://github.com/abap2UI5/abap2UI5/assets/64013300/c163b3cc-bfd7-44df-b419-e7dee8c9946b)
![image](https://github.com/abap2UI5/abap2UI5/assets/64013300/2e535775-b2fb-4adf-8bfb-0a9180fc6c15)
![image](https://github.com/abap2UI5/abap2UI5/assets/64013300/2984675c-f4d6-40b4-82d8-ce562de858ae)
![image](https://github.com/abap2UI5/abap2UI5/assets/64013300/2a76c7c6-1285-45ea-8a86-61447de850fd)

**Comment by oblomov-dev** [comment]
Nice!

**Comment by oblomov-dev** [comment]
i made some changes again to the folder structure, reorganizing the project at the moment, but i hope to finish this process soon. just open a new issue if you run into further problems.
